dashboard-api/test.sh

```diff
@@ -0,0 +1,13 @@
+./stop.sh
+
+echo "\n>>> SETTING UP DATABASE <<<\n"
+docker-compose -f docker-compose-common.yaml -f docker-compose-test.yaml build setup_cockroachdb
+docker-compose -f docker-compose-common.yaml -f docker-compose-test.yaml run setup_cockroachdb
+echo "\n>>> DATABASE IS READY TO RECEIVE CONNECTIONS <<<\n"
+
+echo "\n>>> SETTING UP API <<<\n"
+docker-compose -f docker-compose-common.yaml -f docker-compose-test.yaml build api
+echo "\n>>> FINISHED SETTING UP API <<<\n"
+
+echo "\n>>> RUNNING TEST CASES <<<\n"
+docker-compose -f docker-compose-common.yaml -f docker-compose-test.yaml run api
```

Comment:
If:
- `api` depends on `setup_cockroachdb`, https://github.com/vlab-research/vlab/pull/75/files#diff-fa845e05416b322fd4056e688636b49f1fce19916670dee563a5d479004db617R13
- and `setup_cockroachdb` depends on `cockroachdb`, https://github.com/vlab-research/vlab/pull/75/files#diff-3403c3600e3dae55dc11f60a0884bb3676681159fee1fe90166ad191d2b9da5aR18

Why do we need to _build_ and _run_ `setup_cockroachdb` and `api`? Shouldn't we just _build_ and _run_ `api`?

Reply:
Omg my bad 🤯  good catch !!